src/wallet/walletdb.h

```diff
@@ -93,11 +94,13 @@ class CKeyMetadata
 public:
     static const int VERSION_BASIC=1;
     static const int VERSION_WITH_HDDATA=10;
-    static const int CURRENT_VERSION=VERSION_WITH_HDDATA;
+    static const int VERSION_WITH_KEY_ORIGIN = 12;
```

Comment:
why no 11 :(

Reply:
A previous version of this pr used version 11 and I did not want to cause wallets that had tested that pr to become unusable.

Reply:
From my experience older hww branch wallets are already unusable so I wouldn't worry too much about that :-)